core/src/main/scala/org/apache/spark/streaming/eventhubs/EventHubsClientWrapper.scala

```diff
@@ -131,22 +132,27 @@ private[eventhubs] class EventHubsClientWrapper extends Serializable with EventH
     // Create Eventhubs client
     eventhubsClient = EventHubClient.createFromConnectionStringSync(connectionString)
 
+    // Enabling flag to receive runtime info from eventhub client.
+    val receiverOptions = new ReceiverOptions
+    receiverOptions.setReceiverRuntimeMetricEnabled(true)
```

Comment:
You can calculate by yourself, class header+all fields in runtimeInfo class...so I am thinking if we really want to merge this fix..once their 0.14 is out we can get start seq easily....and all of these can be removed